### PR TITLE
University Campus Hour - Capstone Project 1

### PR DESCRIPTION
**Pull request by PrangonGhose**
- The project has been made using personalized ideas but following [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) for colors, typography, and layout.
- The project is done in plane CSS. No CSS framework has been used for styling.
- Only **Home** and **About** page have been designed as per the requirement of the project.
- Each of the pages have been adopted for 2 different screen sizes - **Mobile** *768px wide* and **Desktop** *768px or wider* as per the requirement.
- Interactions (e.g., *Links* and *Mobile menu*) are put.
- Dynamic page (*Featured Clubs*) is created using JavaScript.
- Readme file has been adopted as per the requirements.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      <div class="adress">
+        <p class="phone-number"><b>Phone:</b> +123456789</p>
+        <p class="adress-details">
+          <b>Address:</b> Lorem, ipsum. <br>
+          Lorem ipsum dolor sit.<br>
+          34220 Lorem, ipsum dolor.
+        </p>
+      </div>
+    </div>
+    <p class="rights"><i class="fa fa-copyright"></i>2021. All rights reserved</p>
```
Comment (Abenezer-Tilahun) [inline about.html]: - [ ]  As stated the project requirement The pages should look almost identical to the original design. So Kindly aligned the   "logo"  the `left` and "text" aligned to the `right` and adjust background color of the footer Just like the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 

The Design Guidelines Look likes this 

Capture


Your Project Design Look likes this 

Capture

File: index.html [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+      <div class="adress">
+        <p class="phone-number"><b>Phone:</b> +123456789</p>
+        <p class="adress-details">
+          <b>Address:</b> Lorem, ipsum. <br>
+          Lorem ipsum dolor sit.<br>
+          34220 Lorem, ipsum dolor.
+        </p>
+      </div>
+    </div>
+    <p class="rights"><i class="fa fa-copyright"></i>2021. All rights reserved</p>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ]  As stated the project requirement The pages should look almost identical to the original design. So Kindly aligned the   "logo"  the `left` and "text" aligned to the `right` Just like the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 

The Design Guidelines Look likes this 

Capture

Your Project Design Look likes this 

Capture

File: about.html [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    <div class="mobile-menu-background">
+      <img src="images/Iconcross-btn.png" alt="Closing button mobile" class="cross-button pointer">
+    </div>
+    <ul class="mobile-menu-list">
+      <li class="mobile-menu-listItems"><a href="about.html">About</a></li>
+      <li class="mobile-menu-listItems"><a href="index.html#main-program">Program</a></li>
+      <li class="mobile-menu-listItems"><a href="index.html#clubs">Clubs</a></li>
+      <li class="mobile-menu-listItems"><a href="#partners">Partners</a></li>
+      <li class="mobile-menu-listItems"><a href="#contact">Contact</a></li>
+    </ul>
```
Comment (Abenezer-Tilahun) [inline about.html]: - [ ] Kindly add "Home" link at the "About page" to  possible to navigate between the pages, there's a link from Home page to About page and the opposite too.

Capture

File: README.md [hunk 4]
```diff
@@ -1 +1,76 @@
...
+## Acknowledgments
+- Original designer [Cindy Shin](https://www.behance.net/adagio07)
+- The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/)
+- [Microverse](https://www.microverse.org/)
+- Code Reviewers
+- Coding Partners
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (Abenezer-Tilahun) [inline README.md]: - [ ] Kindly I recommend you to update your new [README template](https://github.com/microverseinc/readme-template) and All main headers from the README template should be present in a yours repo. But customize according to your project and please Make sure that you added the section "Future Features", including 1-3 potential features to be added.

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @PrangonGhose   , 

Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Professional  README 
✔️ Good commit messages

almost

## Required Changes ♻️

- [ ]  As stated the project requirement you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. it's one of the project requirement you must fulfill for to get approval for project.
Capture

_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by PrangonGhose** [comment]
## Corrections after first review:
- Contact section is fully optimized as per the design guidelines in both about.html and index.html.
- README.md file has been changed as per the new template and future features are added.
- **"Home"** has been added in the mobile menu section.
- Presentation of the project can be found [here](https://www.loom.com/share/151a84b791384da683db56873ea48a41).

File: index.html [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+          <a href="#contact">
+            <li class="hover-effect">Contact</li>
+          </a>
+        </nav>
+        <nav class="union pointer">
+          <img src="images/Union.png" alt="Mobile Menu Option">
+        </nav>
+      </div>
+    </div>
+  </header>
```
Comment (Favourezeugwa) [inline index.html:65]: - Currently, I think the header of your mobile version currently contains the top dark bar and a bottom white bar that is only meant to be shown on your desktop screen. Apparently, from the Behance design, you only need just the **hamburger menu** on the **top left**.

Here is what I mean 👇🏽 
| Your mobile header  |Behance mobile header |
| :---:   | :---: | 
|badmob  |goodmob  |

- Why don't we refactor your mobile header to look just like Behance? 🌸 
You can simply hide them on mobile and show them in your desktop version(using media query)

**Review by Favourezeugwa — CHANGES_REQUESTED** [review]
## STATUS: REQUIRES CHANGES ⚠️ 🚥 ♻️ 
Hi @PrangonGhose , 

Great work on making the changes requested by @Abenezer-Tilahun, your previous reviewer 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


woman so close

 ## Highlights 💯 
- No Linter errors ✔️ 
- Professional Readme file ✔️ 
- Good commit messages ✔️ 
- Descriptive PR ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by PrangonGhose** [comment]
## Corrections after second review:
- The position of hamburger icon has been optimized in the mobile version according to the design guidelines.

**Review by Bondok6 — APPROVED** [review]
## Hi @PrangonGhose 🙋‍♂️, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

- Great job implementing the changes provided by the previous reviewers 💯🎉



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**